### PR TITLE
feat(mobile) Add OAuth Login On Mobile

### PR DESCRIPTION
**Pull request by alextran1502**
OAuth Flow Implementation on Mobile

https://user-images.githubusercontent.com/27055614/202879045-78e2653e-0010-4d15-b481-c3a5ead1f9f9.mp4

### CODE REVIEW

File: mobile/pubspec.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ dependencies:
   latlong2: ^0.8.1
   collection: ^1.16.0
   http_parser: ^4.0.1
+  flutter_web_auth: ^0.5.0
```
Comment (jrasm91) [inline mobile/pubspec.yaml:43]: Did you end up using this dependency?

**Review by jrasm91 — APPROVED** [review]

**Comment by jrasm91** [comment]
Can you add a video of the implementation?

File: mobile/pubspec.yaml (hunk 1 above)
Comment (alextran1502) [reply mobile/pubspec.yaml:43]: Yes I did because the normal web method cannot redirect correctly

**Comment by alextran1502** [comment]
> Can you add a video of the implementation?

@jrasm91 Added in the info section

**Pull request merged**